edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -22,6 +22,8 @@
 import json
 import os
 import yaml
+import glob
+import hashlib
```

Comment:
Please keep dictionary order. (eg. glob should be placed before json after fnmatch.) This is not allowed by lint which is not enabled yet.

Reply:
ack